### PR TITLE
Readme and pagination

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
In this PR, I have added Readme.md and pagination for product listing page

Please review and suggest

### CODE REVIEW

File: src/components/pagination/Pagination.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  };
+
+  const getPaginatedData = () => {
+    const startIndex = currentPage * dataLimit - dataLimit;
+    const endIndex = startIndex + dataLimit;
+    return sortedData.slice(startIndex, endIndex);
+  };
+
+  const getPageNumbersGroup = () => {
+    let start = Math.floor((currentPage - 1) / pageLimit) * pageLimit;
```
Comment (jalajcodes) [inline src/components/pagination/Pagination.jsx]: you can change it to `const`

**Review by jalajcodes — APPROVED** [review]
Looks good Apurva!

File: src/backend/db/products.js [hunk 2]
```diff
@@ -112,28 +112,172 @@ export const products = [
...
     },
     rating:4,
     imgURL:comboSet2,
-    categoryName: "Gifts",
+    categoryName: "Cups",
     offer:"combo-offer",
+    isBestseller: false,
+  },
+  {
+    _id: "8s7079e3-5467-4012-9d58-596887022259",
```
Comment (MayankKumar10) [inline src/backend/db/products.js:127]: instead of that you can use uuid

**Review by MayankKumar10 — APPROVED** [review]
Just a suggestion everything else looks good you can merge it

File: src/backend/db/products.js (hunk 2 above)
Comment (ApurvaSawant11) [reply src/backend/db/products.js:127]: I had used `uuid` before. But a new uuid is generated on page reload. 
Hence, to create single product page feature, I had to change it to fixed uuid.

File: src/components/pagination/Pagination.jsx (hunk 1 above)
Comment (ApurvaSawant11) [reply src/components/pagination/Pagination.jsx]: Sure. Will make the changes

**Pull request merged**